command/genesis/polybft_params.go

```diff
@@ -155,6 +155,14 @@ func (p *genesisParams) generatePolyBftChainConfig(o command.OutputFormatter) er
 		return err
 	}
 
+	governorAdminAddr := types.ZeroAddress
+	if p.governorAdmin != "" {
+		// if no admin is defined, zero address will be the owner,
```

Comment:
Maybe move this comment outside of the if clause. WDYT?

Reply:
Done: https://github.com/0xPolygon/polygon-edge/pull/1728/commits/834a98adb036dc64e915da3b97cc18b4c190aa5a